src/SharpGLTF.Core/Schema2/gltf.MeshPrimitive.CESIUM_primitive_outline.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SharpGLTF.Schema2
+{
+    partial class CESIUM_primitive_outlineglTFprimitiveextension
+    {
+        internal CESIUM_primitive_outlineglTFprimitiveextension(MeshPrimitive meshPrimitive) { }
+
+        public int? Indices
+        {
+            get => _indices;
+            set => _indices = value;
+        }
+    }
+    partial class MeshPrimitive
+    {
+        public void SetCesiumOutline(Accessor accessor)
+        {
+            if (accessor == null) { RemoveExtensions<CESIUM_primitive_outlineglTFprimitiveextension>(); return; }
+
+            Guard.NotNull(accessor, nameof(accessor));
+            Guard.MustShareLogicalParent(LogicalParent.LogicalParent, "this", accessor, nameof(accessor));
+            Guard.IsTrue(accessor.Encoding == EncodingType.UNSIGNED_INT, nameof(accessor));
+            Guard.IsTrue(accessor.Dimensions == DimensionType.SCALAR, nameof(accessor));
+
+            var ext = UseExtension<CESIUM_primitive_outlineglTFprimitiveextension>();
+            ext.Indices = accessor.LogicalIndex;
+        }
+    }
+
+    public static class CesiumOutline
+    {
+        public static Accessor CreateCesiumOutlineAccessor(ModelRoot model, IReadOnlyList<uint> outlines)
```

Comment:
Note: I've also added ModelRoot as parameter (in addition to IReadOnlyList) because it makes creating the bufferView and Accessor more easy

Reply:
Hmm....  Given that Cesium has a bunch of extension, it's possible in the future other cesium related methods will be added, so I would rename "CesiumOutline" to "CesiumUtils" , "CesiumToolkit" "CesiumExtensions"  , whatever you like more.

That way, all future cesium related methods can go in the same place.

Reply:
okay, renamed it to CesiumToolkit